README.md

```diff
@@ -186,6 +186,33 @@ curl http://127.0.0.1:5000/chat \
 {"response":" I'm a software developer with a passion for building innovative and user-friendly applications. I have experience in developing web and mobile applications using various technologies such as Java, Python, and JavaScript. I'm always looking for new challenges and opportunities to learn and grow as a developer.\n\nIn my free time, I enjoy reading books on computer science and programming, as well as experimenting with new technologies and techniques. I'm also interested in machine learning and artificial intelligence, and I'm always looking for ways to apply these concepts to real-world problems.\n\nI'm excited to be a part of the developer community and to have the opportunity to share my knowledge and experience with others. I'm always happy to help with any questions or problems you may have, and I'm looking forward to learning from you as well.\n\nThank you for visiting my profile! I hope you find my information helpful and interesting. If you have any questions or would like to discuss any topics, please feel free to reach out to me. I"}
 ```
 
+In the request, setting `stream` to "true" emits a response in chunks. Currently, this response
+is plaintext and not formatted to the OpenAI API specification.
+```
+curl http://127.0.0.1:5000/chat \
+  -H "Content-Type: application/json" \
+  -d '{
+    "model": "stories15M",
+    "stream": "true",
```

Comment:
Can we do this with llama3 and collapse the 2 code snippets?

So 
<Curl with stream=false>
<output with false>
> In the request, setting `stream` to "true" emits a response in chunks. Currently, this response
is plaintext and not formatted to the OpenAI API specification
<output with true>

Reply:
The output would be exactly the same whether streamed or not. For now I"ll just remove both new code blocks and just set stream explicitly in the arguments.